src/components/MintSong/MintSongModal/MintForm/MintForm.types.ts

```diff
@@ -1,9 +1,11 @@
+import { UploadFile } from "antd"
 export type MintMusicWAdFormValues = {
```

Comment:
isn't `MintMusicWAdFormValues` supposed to be `MintMusicAdFormValues`? what is the extra `W` supposed to mean?

Reply:
Bro it is not introduced in this PR, create new issue instead

Reply:
Also `W` mean with, so that is `MintMusic(With)AdFormValues`

Reply:
please be explicit with your namings Om. You know there is a slim chance anyone would understand what the W means.

You can leave it for now, we can fix it later.

Reply:
Tracked here https://github.com/meta-web3-music/frontend/issues/92